bindings/ruby/extconf.rb

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/env ruby
+
+require 'mkmf'
+
+$LDFLAGS << ' -lc++'
```

Comment:
> the whole bundle of implicit flags turned on by that

What are those? Are they flags we would want anyways when linking a C++ library?